content/millennium/bulk-data/group-export.md

```diff
@@ -0,0 +1,296 @@
+
+## Status Request
+
+Checks the status of the export job.
+Use the URL returned in the `Content-Location` header from the kick off request to retrieve the status of a bulk data export:
+
+    GET  <URL from Content-Location header>
+
+_Notes_
+
+* Clients should follow an Exponential Backoff approach when polling for status as outlined in the [bulk data export specification.](https://hl7.org/fhir/uv/bulkdata/STU1.0.1/export/index.html#bulk-data-status-request)
+
+### Authorization Types
+
+<%= authorization_types(provider: false, patient: false, system: true) %>
+
+_Notes_
+
+* Scopes are not checked in this request. The authorization token must be valid, and the client in the token must match the client that requested the export.
+
+### Headers
+
+<%= headers head: {'Accept': 'application/fhir+json', Authorization: '&lt;OAuth2 Bearer Token>'} %>
+
+### Example
+
+#### In-Progress Response
+
+Request
+
+    GET https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/bulk-export/jobs/<job_id>
+
+Response
+
+    Status: 202 Accepted
+    X-Progress: in-progress
+    Retry-After: 600
+
+#### Complete Response
+
+Request
+
+    GET https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/bulk-export/jobs/<job_id>
+
+Response
+
+    Status: 200 OK
+    Expires: Mon, 22 Jul 2025 23:59:59 GMT
+    Content-Type: application/json
+
+<%= json(:R4_BULK_EXPORT_COMPLETE) %>
+
+<%= disclaimer %>
+
+### Errors
+
+The common [errors] and [OperationOutcomes] may be returned.
+
+## File Retrieval Request
+
+Once an export is complete, the status request will return file URLs which may be used to download the exported resources.
+
+    GET <File URL from status response>
+
+_Implementation Notes_
+
+* The file retrieval request returns a '307 Temporary Redirect' with a 'Location' header, the Location header must be followed in order to download the file
+* An exported ndjson file contains only a single resource type and there can be multiple files for the same resource type. There is no limit number of resources per file.
+* Files should be retrieved promptly, as they will expire after 30 days.
+* The initial file retrieval request requires the Authorization header to be sent, when following the re-direct, it is no longer appropriate. Ensure the Authorization header is not perpetuated.
+
+_Retrieval of Attachments_
+
+* [Binary](/millennium/r4/foundation/other/binary) resources are not returned by bulk data operations.
+* Attachment references must be retrieved separately using the [Binary](/millennium/r4/foundation/other/binary) resource URL from the DocumentReference.content.attachment and DiagnosticReport.presentedForm fields from the export files.
+
+### Authorization Types
+
+<%= authorization_types(provider: false, patient: false, system: true) %>
+
+_Notes_
+
+* The authorization token must contain the system read scope for the resource that the export file contains.
```

Comment:
would you kind adding the info that the client requesting  the file matches the client who created the export?

Reply:
Added here: https://github.com/cerner/fhir.cerner.com/pull/899/commits/c936d35a3ba46f521cf0e5f6addfec2b7276c599#